### ISSUE TITLE
⭐️ Golden Edition - Add “Close Tabs” Option to DevTools Right-Click Menu

### ISSUE THREAD

**Issue by swagfin** [usebruno/bruno#3368]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

## Feature Request

It would be helpful if the DevTools tabs had a right-click menu similar to the one for request tabs, with options such as:

    Close
    Close Others
    Close to the Left
    Close to the Right
    Close All Tabs

Currently, closing multiple DevTools tabs requires manually closing each tab individually. Adding these options would streamline the process, especially for users working with multiple tabs in DevTools.

### Benefit
This feature would improve productivity and enhance user experience, allowing for quick tab management without the need for individual closures.

### Mockups or Images of the feature

![image](https://github.com/user-attachments/assets/bc4df921-d2d9-4d89-8247-88af7a22b00c)

Request 
![image](https://github.com/user-attachments/assets/aa738063-bb7a-456f-8584-c1776bfa2412)

**Comment by helloanoop** [comment]
Hey @swagfin !

Thank you for the suggestion. I completely agree with the benefits of adding the additional menus. I've added it to the backlog and will prioritize it once we've addressed some of the critical issues the team is currently focused on.

**Comment by james-ha-bruno** [comment]
I don't believe this section and ticket are applicable anymore, should this be closed @sreelakshmi-bruno ?
